tests/phpunit/unit-tests/class-llms-test-main-class.php

```diff
@@ -27,15 +28,14 @@ public function setUp() {
 	}
 
 	/**
-	 * test the _instance variable
+	 * Test the `instance` property.
 	 *
 	 * @since 3.3.1
-	 *
-	 * @return void
```

Comment:
are we deciding to remove the return type from tests? Or only when it's void?

Reply:
We haven't decided anything like this

Reply:
Restore this line

Reply:
From the [WP PHP documentation standard](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#1-functions-class-methods):

> @return void should not be used outside of the default bundled themes.

Also, the [phpDocumentor](https://docs.phpdoc.org/3.0/guide/references/phpdoc/tags/return.html) site:

> functions and methods without a `return` value, the @return tag MAY be omitted here, in which case @return void is implied.